README.md

```diff
@@ -69,5 +49,3 @@ Give a ⭐️ if you like this project!
 ## 📝 License
 
 This project is [MIT](./LICENSE) licensed.
-
-_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
```

Comment:
- [ ] OPTIONAL
MIT license link is broken, kindly endeavor to fix it, and please ensure to remove redundant instructions from the original template. Implementing this changes will make your ReadMe appear more professional.